libcontainer/cgroups/fs2/memory.go

```diff
@@ -219,3 +227,147 @@ func statsFromMeminfo(stats *cgroups.Stats) error {
 
 	return nil
 }
+
+func getPageUsageByNUMAV2(path string) (cgroups.PageUsageByNUMA, error) {
```

Comment:
nit: no need to have V2 in the function name, this package only contains cgroup v2 code.

Reply:
Got it.